src/Region.h

```diff
@@ -164,8 +165,8 @@ class Halfspace : public Region {
 class RectangularPrism : public Intersection {
 
 public:
-  RectangularPrism(double width_x, double width_y,
-		   double origin_x=0., double origin_y=0.);
+  RectangularPrism(double width_x, double width_y, double width_z=1.,
```

Comment:
width_z default value should be inf or a large value like 1.0E100. I suppose the majority use of this function is to define 2D rectangle, so it is better to change the order of parameters avoiding the necessity to write width_z when change origin_x origin_y. Like this
  RectangularPrism(double width_x, double width_y, 
                   double origin_x=0., double origin_y=0., double width_z=1E100.,double origin_z=0.);

Reply:
Makes sense!
I'll change it